### PR TITLE
Style notifications

### PR DESCRIPTION
**Pull request by kennyt**
px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/js/mixins/TabsMixin.js [hunk 1]
```diff
@@ -96,25 +98,33 @@ const TabsMixin = {
...
       return (
-        <span className="badge-container badge-primary">
-          {link}
-          <span className="badge text-align-center">{notificationCount}</span>
-        </span>
+        <Link
+          to={tab}
+          className={tabLabelClass}
+          onClick={this.tabs_handleTabClick.bind(this, tab)}>
+          <span className="tab-label-text">
```
Comment (jfurrow) [inline src/js/mixins/TabsMixin.js]: I think this classname should actually be `tab-item-label-text` to follow the parent->children relationship (e.g. the parent is `tab-item`, then `tab-item-label`, and here `tab-item-label-text`)

Comment (kennyt) [inline src/js/mixins/TabsMixin.js]: done

**Comment by jfurrow** [comment]
Sorry for removing my label — I want to test this because `inline-flex` has spotty IE11 support.

**Comment by jfurrow** [comment]
Looks great in IE!

**Pull request merged**